src/event/block/BlockSproutEvent.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+declare(strict_types=1);
+
+namespace pocketmine\event\block;
+
+use pocketmine\block\Block;
+use pocketmine\event\Cancellable;
+use pocketmine\event\CancellableTrait;
+use pocketmine\world\BlockTransaction;
+
+/**
+ * Called when Saplings or Bamboo grow.
+ * These types of plants tend to change multiple blocks at once upon growing, that's why this event returns the BlockTransaction.
```

Comment:
```suggestion
 * These types of plants tend to change multiple blocks at once upon growing.
```
New users aren't going to question the fact that it returns a BlockTransaction.